user-stories/US-08-Data-Consumer-interface.md

```diff
@@ -0,0 +1,113 @@
+
+**because** I need to organize the list in way that it is easier for me to process all Privacy requests in legal due time
+
+## US-08-04
+
+**As a** Data Consumer,
+
+**I want to** clearly see the deadline by which I have to legally reply to a Privacy request 
+
+**because** I want to stay out of any legal trouble and process all the Privacy requests I receveice in the legal due time
+
+## US-08-05
+
+**As a** Data Consumer,
+
+**I want to** view the details of each Privacy request I received (date of the request, Submitter identification, type of request and details of request (data category, processing catgory, target, provenance...), additional elements if there are any such as identification document, message, data range, data capture, data reference, selected data to delete, selected data to modify, new data modified, consent restriction...)
+
+**because** I need to take notice of them to proccess the Privact request
+
+## US-08-06
+
+**As a** Data Consumer,
+
+**I want to** be able to see and downlaod the details of an old Privacy request : (date of the request, Submitter identification, type of request and details of request (data category, processing catgory, target, provenance...), additional elements if there are any such as identification document, message, data range, data capture, data reference, selected data to delete, selected data to modify, new data modified, consent restriction...), type of answer (approved/partially approvec/denied), date of the answer, date of the action associated with the type of request (e.g user has accessed data)
+
+**because** I need to keep track of them for legal reason
+
+## US-08-07
+
+**As a** Data Consumer,
+
+**I want to** process the Privacy requests, that can't be automatically processed, directly from the interface
+
+**because** it is not convenient for me to manually write all an email and manually fetch all elements to answer the Privacy request
+
+## US-08-08
+
+**As a** Data Consumer,
+
+**I want to** be able to track the Privacy requests that are automatically processed
+
+**because** I need to be aware of them and to keep track of them for legal reason
+
+## US-08-09
+
+**As a** Data Consumer,
+
+**I want to** be able to reply to a Privacy request by also adding a message
+
+**because** I might have to deny a Privacy request and in that case I must give a specific reason to the submitter
+
+## US-08-10
+
+**As a** Data Consumer,
+
+**I want to** be guided if I need to add a message to specifiy why I must deny a Privacy Request
+
+**because** I want to save time and I might not know what are the valid reasons to deny a Privacy request
+
+## US-08-11 
+
+**As a** Data Consumer,
+
+**I want to** be able easily access and send a Privacy request capture link 
+
+**because** a submitter might directly ask me where they can submit a Privacy Request 
+
+## US-07-08-12 - other (not part of interface per se)
```

Comment:
@Clementinev Do we plan to write user stories for the [privacy computation engine](https://github.com/blindnet-io/product-management/tree/main/refs/high-level-architecture#privacy-computation-engine)? Specifically the privacy compiler. If so, would US-07-08-12 and 13 make more sense there?

Reply:
IMHO, Privacy Computation Engine (and especially Privacy Compiler) is not a user-facing component and as such is not concerned by user stories. Those compoments aly abide to Functional Requeirements, and other noramtive doduments.

Reply:
@milstan I was thinking in our case with building a devkit, the "users" can be developers too, who might be using the privacy computation engine by itself. I see what you mean though, we typically write stories with "end-users" in mind.

Reply:
Also, i think our current approach to documentation is by every measure redundant. 

We have high-level/product normative specifications, user stories, issues/functional requirements. And we call ourselves agile :)

The same information is currently present in at lest, more then one place. 

That is another reason why I consider we don't need the user stories for PC.